packages/@aws-cdk/aws-autoscaling/README.md

```diff
@@ -488,6 +488,16 @@ const aspect = new autoscaling.AutoScalingGroupRequireImdsv2Aspect();
 Aspects.of(this).add(aspect);
 ```
 
+## Warm Pool
+
+Auto Scaling offers [a warm pool](https://docs.aws.amazon.com/autoscaling/ec2/userguide/ec2-auto-scaling-warm-pools.html) which gives an ability to decrease latency for applications that have exceptionally long boot times. You can create a warm pool with default parameters as below:
+
+```ts
+declare const autoScalingGroup: autoscaling.AutoScalingGroup;
+
+autoScalingGroup.addWarmPool('Pool', {});
```

Comment:
```suggestion
autoScalingGroup.addWarmPool('Pool');
```

Reminder that when we change the `addWarmPool` signature this should change as well. I think it might be beneficial to have a second example of `addWarmPool` to show what optional parameters you can set.

Reply:
Sure, will do.